apis/kubedb/v1alpha2/mysql_helpers.go

```diff
@@ -372,6 +372,10 @@ func (m *MySQLSpec) GetPersistentSecrets() []string {
 	if m.AuthSecret != nil {
 		secrets = append(secrets, m.AuthSecret.Name)
 	}
+
+	secrets = append(secrets, MsSQLEndpointCertsSecretName)
+	secrets = append(secrets, MsSQLDbmLoginSecretName)
+	secrets = append(secrets, MsSQLMasterKeySecretName)
```

Comment:
Why is it here? Shouldn't it be in MSSQL Spec?

Reply:
Ooh!